website/static/css/registration_landing.css

```diff
@@ -0,0 +1,146 @@
+/* PREREG PAGES */
```

Comment:
Drive by: It looks like this file was copied from (or shares a lot of code with) `prereg.css`. Can this be DRY'd up?